downstream/assemblies/platform/assembly-upgrade-faqs.adoc

```diff
@@ -2,4 +2,63 @@
+
+What is the difference between "upgrade" and "migration" in {PlatformNameShort} 2.6?::
+An upgrade is an application action, such as updating {PlatformNameShort} 2.5 to 2.6. 
+A migration involves moving data, such as from an RPM-based 2.6 installation to a container-based 2.6 installation. 
+New service components are not explicitly required between versions 2.5 and 2.6.
+
+Can I upgrade from 2.4 to 2.6 or must I upgrade to 2.5 first?::
+Yes you can upgrade directly to 2.6. 
+However note that there may be new system requirements you will need to update before upgrading. 
+
+How will managed cloud customers be upgraded to {PlatformNameShort} 2.6?::
+All managed cloud customers on {Azure} and {AWS} will be upgraded to 2.6. 
+Two upgrade window options will be available following the platform upgrade: non-production or production environments.
+Communications will be ongoing from mid-July until late September.
+
+Will migrations be fully supported in {PlatformNameShort} 2.6?::
+Yes, migrations are fully supported, enabling customers to transition from RPM installations to containerized or {OCPShort} environments, or to the managed offering. 
+Customers must be on the latest version of {PlatformNameShort} for their current installation prior to migrating. 
+The installer will manage new components introduced in version 2.5 for direct 2.4 to 2.6 upgrades.
+
+//Need to add link to section when it exists. 
+//When upgrading from 2.4 to 2.6 (applies only to RPM or {OCPShort}), what’s different about the upgrade process compared with the 2.4 to 2.5 process?::
+//See the Overview of upgrade improvements section.
+
+I’m using {EDAName} (EDA) in 2.4, can I upgrade EDA to 2.6?::
+If you are using {PlatformNameShort} 2.4 with the technical preview of EDA controller but want to upgrade to the {PlatformNameShort} 2.6 with EDA, you must install a new instance of {PlatformNameShort} 2.6 and manually re-create your EDA configurations in the new, fully integrated environment.
+
+Will my existing 2.4 or 2.5 OAuth Applications/Tokens, Credentials/Customer Credentials, and Personal Access Tokens still work after upgrading to 2.6?::
+For upgrades from {PlatformNameShort}  2.4 or 2.5 to {PlatformNameShort}  2.6, some manual configuration is required:
+* OAuth applications:
+** {ControllerNameStart}: You can view and edit existing {ControllerName} applications, but you cannot create new ones. They still function, but they might be removed in a future release. You should plan to migrate to platform OAuth applications.
+** {PlatformNameShort}: Platform OAuth applications provide an updated interface and are the standard for future use. You will transition to these applications.
+* Tokens:
+** {ControllerNameStart}: {ControllerName} personal access tokens (PATs) are deprecated. You will move to platform gateway PATs.
```

Comment:
Suggestion: Since you're using the controller name at the start of the sentence, maybe continue using the {ControllerNameStart} here. 
```suggestion
** {ControllerNameStart}: {ControllerNameStart} personal access tokens (PATs) are deprecated. You will move to platform gateway PATs.
```